rplugin/python3/deoplete/sources/deoplete_jedi/cache.py

```diff
@@ -393,16 +414,19 @@ def cache_context(filename, context, source, extra_path):
             import_key = import_key.rstrip('.')
             module_file = utils.module_search(
                 import_key,
-                chain(extra_path,
-                      [context.get('cwd'), os.path.dirname(filename)],
-                      utils.rplugin_runtime_paths(context)))
+                chain(
+                    extra_path,
+                    [context.get('cwd'), os.path.dirname(filename)],
+                    utils.rplugin_runtime_paths(context)
+                )
+            )
             if module_file:
                 cache_key = (import_key, 'local')
                 extra_modules.append(module_file)
             elif is_package(import_key):
                 cache_key = (import_key, 'package')
             elif not cinput.endswith('.'):
-                cache_key = ('import~',)
+                cache_key = ('import~', )
```

Comment:
Are those spaces for tuples with just one element "required" - or just done automatically?
I am -0 on those changes.